Assets/Scripts/Menu/HighScores.cs

```diff
@@ -17,8 +20,7 @@ void Start()
             {
                 temp = temp + (i+1) + ". " + topScores[i] + "\n";
             }
-            highScoreText.text = temp;
+            GameObject.Find("HighScores").GetComponent<TMP_Text>().text = temp;
```

Comment:
![image](https://user-images.githubusercontent.com/32081318/179624479-ff43b5e7-a676-47a0-8560-4c6e485676dc.png)
Getting a null reference exception when navigating back from the Settings screen.

Reply:
GameObject.Find by name doesn't work if the object isn't active at the time the method is called. Since this is called on the back button of Settings when HighScores isn't active, it isn't finding anything.

A better way to get the object is to set a public highScores game object in HighScores.cs, then directly drag and assign the HighScores object into the highScores field in the script component of Scores. If you have questions I can help.

Reply:
Wait nvm you had this before, you just had to move the code to its own UpdateScores method and called it